libs/features/mas/docs/common.js

```diff
@@ -0,0 +1,40 @@
+import '../../../deps/custom-elements.js';
```

Comment:
maybe its better to place it in `src` folder
and keep docs folder for html build artifacts